androidsvg/src/main/java/com/caverock/androidsvg/utils/RenderOptionsBase.java

```diff
@@ -90,47 +91,30 @@ public RenderOptionsBase(RenderOptionsBase other)
    }
 
    /**
-    * Specifies some additional CSS rules that will be applied during render in addition to
-    * any specified in the file itself.
+    * Specifies some already parsed CSS that will be applied during render in
+    * addition to any specified in the file itself.
     * @param css CSS rules to apply
     * @return this same <code>RenderOptions</code> instance
     */
-   public RenderOptionsBase css(String css)
+   public RenderOptionsBase css(CSS css)
    {
-      this.css = css;
+      this.cssRuleset = css.cssRuleset;
       return this;
    }
```

Comment:
I don't think we should have two independent sets of CSS.  Setting one should override the other.  Same as if you did

    .css(styles1).css(styles2)